Monika After Story/game/zz_backgrounds.rpy

```diff
@@ -2146,10 +2146,17 @@ init -10 python:
                     tag definition for this to work.
             """
             if tag is not None:
-                adv_frame = self.get_deco_adf(tag)
-                deco = store.mas_deco.get_deco(tag)
+                deco_info = self.get_deco_info(tag)
+                if deco_info is not None:
+                    real_tag, adv_frame = deco_info
+
+                deco = store.mas_deco.get_deco(real_tag)
                 if adv_frame is not None and deco is not None:
-                    self._deco_man._adv_add_deco(deco, adv_frame)
+                    self._deco_man._adv_add_deco(
+                        deco,
+                        adv_frame,
+                        override_tag=tag
+                    )
```

Comment:
I got an unbound error here, let's put it under the previous if check (github won't let me add a suggestion).
```renpy
            if tag is not None:
                deco_info = self.get_deco_info(tag)
                if deco_info is not None:
                    real_tag, adv_frame = deco_info

                    deco = store.mas_deco.get_deco(real_tag)
                    if adv_frame is not None and deco is not None:
                        self._deco_man._adv_add_deco(
                            deco,
                            adv_frame,
                            override_tag=tag
                        )
```